instrumentation/pekko/pekko-actor-1.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/pekkoactor/v1_0/ExecutionContextWrapper.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.pekkoactor.v1_0;
+
+import io.opentelemetry.context.Context;
+import scala.concurrent.ExecutionContext;
+
+public class ExecutionContextWrapper implements ExecutionContext {
```

Comment:
usually we make such classes final

Reply:
I was able to remove this class in latest commit. I prefer not to have to extend Scala classes because of the fact that they might differ across Scala versions.